.github/workflows/ci.yml

```diff
@@ -1093,6 +1134,8 @@ jobs:
       # Basic
       - build_all
       - test_all
+      - build_all_arm64
+      - test_all_arm64
```

Comment:
I think these should move down to the `# Platforms` group, next to `build_test_all_macos_arm64` etc.